src/content/guides/deploying-istio/index.md

```diff
@@ -0,0 +1,283 @@
+
+__Warning:__ If your apps have a defined a customer service account then you need to create the binding for each of them.
+
+```
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: istio
+spec:
+  privileged: true
+  allowedCapabilities:
+  - 'NET_ADMIN'
+  fsGroup:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  volumes:
+  - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: istio
+rules:
+- apiGroups:
+  - extensions
+  resourceNames:
+  - istio
+  resources:
+  - podsecuritypolicies
+  verbs:
+  - use
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: istio
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: istio
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: <NAMESPACE_OF_YOUR_SERVICES>
+```
+
+## Recomendations 
+
+The deployment is made up of a different number of components. Some of them, as pilot, have a large impact in terms of memory and CPU, so it is recommended to have around 8GB and 4CPUs free in your cluster. Obviously, all components have `requested resources` defined, so in case you don't have enough capacity you will see pods not starting.
+
+A followed convention, not only within Istio community but Kubernetes too, is to define in all your app deployments the label `app` and `version` as metadata. Both labels help the tracing and metric system to have richer meta information. 
+
+Another good practice is to name the service ports. Istio uses the name to discover the protocol used by the end service container. Thus you have to prefix the port name with the protocol desired. The different supported protocols (http, http2, grpc, mongo, or redis) leverage Istio to route traffic more intelligently. Otherwise, it will use plain TCP routing.
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: app
+spec:
+  ports:
+  - name: http-app
+    port: 80
+    protocol: TCP
+    targetPort: 80
+```
+
+In the strange case, you use multiple services laying over the same pods, you have to use the same kind of traffic for the ports defined in the service.
+
+```
+# WRONG because the protocol is different and same port
+apiVersion: v1             apiVersion: v1
+kind: Service              kind: Service
+metadata:                  metadata:
+  name: app-svc-1            name: app-svc-2
+spec:                      spec:
+  ports:                     ports:
+  - name: http-app           - name: http2-app
+    port: 80                   port: 80
+    protocol: TCP              protocol: TCP
+  selector:                  selector:    
+    app: my-app                app: my-app
+```
```

Comment:
This code block doesn't make sense to me. Is one right and one wrong? Why are the next to each other?

Reply:
yeah I wanted to add an example of the wrong configuration to make clear my point. Here, I exposed two services pointing same app (selector matches) but using different protocols

Reply:
It took time to me for understanding (because Istio docs are short in this matter) and I wanted to give more clues 

Ideas welcome

Reply:
I showed them in columns to make clearer the difference. But it looks it does not work 😓

Reply:
How likely is this to happen BTW? Should this just be an item in the Troubleshooting section? And I think the example is not needed here, as it's not illustrating a lot.

Reply:
ok I like the idea of moving it to the Troubleshooting

Reply:
done